portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Comment {
+
+    // Name of user leaving the comment
+    private String name;
+
+    // Location of user
+    private String location;
+
+    // Text content of comment
+    private String content;
```

Comment:
I think that if you named the variables user_name (or userName), user_location (userLocation) and kept content, it would not be necessary to have the comments as the names would be clear enough about what the content is. The only exception might be what form you expect location to be. Is this country, province, street address?
That is where the comments are useful. In describing more of the meaning or content of the fields than is easily included in the name.

This would also mean changing the accessors to getUserName, etc.